sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,14 +116,14 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 22962057df1b9d538e08088a7b75d9d8e7c29f90 (HEAD, origin/main, origin/HEAD)
-  # Merge: e824ddc2 f0a1c433
-  # Author: aarongreig <aaron.greig@codeplay.com>
-  # Date:   Fri Sep 27 16:54:04 2024 +0100
-  #     Merge pull request #2017 from nrspruit/new_sysman_init
-  #     [L0] Use zesInit for SysMan API usage
-  set(UNIFIED_RUNTIME_TAG 22962057df1b9d538e08088a7b75d9d8e7c29f90)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/Seanst98/unified-runtime.git")
+  # commit d0a50523006fa6f283da6a36811081add3bb22fc
+  # Merge: 804851e4 04deb8b3
+  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
+  # Date:   Tue Aug 20 16:28:30 2024 +0100
+  #    Merge pull request #1940 from RossBrunton/ross/urcall
+  #    [XPTI] Use `ur.call` rather than `ur` in XPTI
+  set(UNIFIED_RUNTIME_TAG bcf2244dccdef352afaf4d4520526573876981e3)
```

Comment:
https://github.com/oneapi-src/unified-runtime/pull/2056 is merged, please update the tag

Reply:
I've updated the tag. Let me know if it doesn't look good.